Vertical tab bar

IbrahimaSal opened the pull request.

EliTheSurfer commented on src/components/App.tsx:
```diff
@@ -1,12 +1,105 @@
+const MenuList:ISideBarComponent[] = [HomeSideBarElement,
+  SettingsSideBarElement, DashBoardSideBarElement, AboutSideBarElement, BlockChainSideBarElement];
+
+function App() : ReactElement {
```
this is the only thing that should be here

EliTheSurfer commented on src/components/App.tsx:
```diff
@@ -1,12 +1,105 @@
+    </div>
+  );
+}
+const HomeSideBarElement : ISideBarComponent = {
```
do a component file for every little element. this should not be here

EliTheSurfer commented on src/components/SideBar.tsx:
```diff
@@ -0,0 +1,73 @@
+  const [visible, setVisible] = useState<boolean>(false);
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.ChangeEvent<unknown>, newValue: number) => {
```
you shouldn't use unknown type

EliTheSurfer commented on src/index.css:
```diff
@@ -1,13 +1,4 @@
-    monospace;
+body{
+  background-color: bisque;
+  padding-left: 0%;
```
why? this shouldn't be there

IbrahimaSal committed:
vertical tab bar

vertical tab bar

sidebar

vertical

verticaltabbar corrected

verticaltabbar revisited

verticaltabbar revisited

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

navbar fixed

aws test

aws test

aws test

aws test

ALL OK

blockchain designed

blockchain designed

blockchain designed

appbarfixed

persistent sidebar

persistent sidebar

persistent sidebar

persistent sidebar
